hazelcast-sql/src/test/resources/META-INF/services/com.hazelcast.datalink.DataLinkRegistration

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+com.hazelcast.jet.sql.impl.ShowStatementTest$DataLinkWithResourcesRegistration
```

Comment:
minor: new line at the end

Reply:
Actually, this is a leftover file - removed

2463c5d1464